File: pset6/mario.py
from cs50 import get_int

# prompt valid input
height = 0;
while True:
    height = get_int("Height: ")
    if height in range(1, 9):
        break

# output loop
for i in range(1, height+1):
    for j in range(height, i, -1):
        print(" ", end="")
    for k in range(i):
        print("#", end="")
    print("  ", end="")
    for l in range(i):
        print("#", end="")
    print("")

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // prompt valid input
    do
        int height = get_int("Height: ");
    while (height < 1 || height > 8);

    // output loop
    for (int i = 0; i < height; i++)
    {
        for (int j = height; j > i + 1; j--)
            printf(" ");
        for (int k = 0; k < i + 1; k++)
            printf("#");
        printf("  ");
        for (int l = 0; l < i + 1; l++)
            printf("#");
        printf("\n");
    }
}